container/libcontainer/handler.go

```diff
@@ -414,6 +414,8 @@ func udpStatsFromProc(rootFs string, pid int, file string) (info.UdpStat, error)
 		return udpStats, fmt.Errorf("couldn't read udp stats: %v", err)
 	}
 
+	r.Close()
```

Comment:
can we move this to just after we handle the error from os.Open, and `defer` it?  That way we ensure it is called if scanUdpStats(r) returns an error as well.